chat/input-bar/src/InputBar/InputBar.tsx

```diff
@@ -336,6 +342,20 @@ export const InputBar = forwardRef<HTMLFormElement, InputBarProps>(
       isOpen && withTypeAhead,
     );
 
+    useEventListener('keydown', (e: KeyboardEvent) => {
+      if (
+        !e.repeat &&
+        e.key === '/' &&
+        shouldRenderHotkeyIndicator &&
+        !isFocused &&
+        textareaRef.current
+      ) {
```

Comment:
Just fyi, had to keep the `textareaRef.current` check inside the hook body because of a failing test -- I think the component wasn't being re-rendered when the ref attached resulting in the hook being out-of-date (i.e. `enabled` stayed false and it wasn't picking up any keyboard inputs when it should have been)